foundation/foundation-mda/src/main/resources/templates/data-delivery-common/pipeline.chart.pom.configuration.vm

```diff
@@ -1,70 +1,70 @@
-                    <argument>template</argument>
-                    <argument>${aissemble.helm.repo.protocol}://${aissemble.helm.repo}/aissemble-spark-application-chart</argument>
-                    <argument>--version</argument>
-                    <argument>${version.aissemble}</argument>
-                    <argument>--values</argument>
-                    <argument>${mainValuesPath}/${moduleArtifactId}-base-values.yaml,${mainValuesPath}/${moduleArtifactId}-dev-values.yaml</argument>
-                    <argument>-s</argument>
-                    <argument>templates/deployment.yaml</argument>
-                </arguments>
-                <outputFile>target/apps/${moduleArtifactId}-dev-chart.yaml</outputFile>
-            </configuration>
-        </execution>
-        <execution>
-            <id>test-chart</id>
-            <phase>compile</phase>
-            <goals>
-                <goal>exec</goal>
-            </goals>
-            <configuration>
-                <executable>helm</executable>
-                <arguments>
-                    <argument>template</argument>
-                    <argument>${aissemble.helm.repo.protocol}://${aissemble.helm.repo}/aissemble-spark-application-chart</argument>
-                    <argument>--version</argument>
-                    <argument>${version.aissemble}</argument>
-                    <argument>--values</argument>
-                    <argument>${mainValuesPath}/${moduleArtifactId}-base-values.yaml,${testValuesPath}/${moduleArtifactId}-test-values.yaml</argument>
-                    <argument>-s</argument>
-                    <argument>templates/deployment.yaml</argument>
-                </arguments>
-                <outputFile>target/apps/${moduleArtifactId}-test-chart.yaml</outputFile>
-            </configuration>
-        </execution>
-        <execution>
-            <id>ci-chart</id>
-            <phase>compile</phase>
-            <goals>
-                <goal>exec</goal>
-            </goals>
-            <configuration>
-                <executable>helm</executable>
-                <arguments>
-                    <argument>template</argument>
-                    <argument>${aissemble.helm.repo.protocol}://${aissemble.helm.repo}/aissemble-spark-application-chart</argument>
-                    <argument>--version</argument>
-                    <argument>${version.aissemble}</argument>
-                    <argument>--values</argument>
-                    <argument>${mainValuesPath}/${moduleArtifactId}-base-values.yaml,${mainValuesPath}/${moduleArtifactId}-ci-values.yaml</argument>
-                    <argument>-s</argument>
-                    <argument>templates/deployment.yaml</argument>
-                </arguments>
-                <outputFile>target/apps/${moduleArtifactId}-ci-chart.yaml</outputFile>
-            </configuration>
-        </execution>
-    </executions>
-</plugin>
\ No newline at end of file
+            <plugin>
```

Comment:
I :Fixed tabbing issue in downstream project poms where this whole block was not being indented